app/cdap/components/GridTable/index.tsx

```diff
@@ -237,16 +246,23 @@ export default function GridTable() {
     getGridTableData();
   }, [gridData]);
 
+  const setSnackbarState = (value: boolean) => {
+    setSnackbar((prevState) => ({
```

Comment:
why is the usage of this different than the other file? you didn't use previous state in the other file